users-guide/english/README.md

```
@@ -0,0 +1,31 @@
+
+## For contributors
+
+### Requirements
+
+* Install [Just](https://github.com/casey/just) 
+```bash
+cargo install just
+```
+### Procedures
+
+**Init**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)